cmd/init.go

```diff
@@ -321,13 +321,12 @@ func install(config *initCommandConfig) error {
 	if config.StackRegistryInit != "" {
 		config.Debug.Log("The flag --stack-registry was set to: ", config.StackRegistryInit)
 		stackRegistry = config.StackRegistryInit
-	} else if configFileStackRegistry == "" {
```

Comment:
I would suggest changing this line to
```
else if configFileStackRegistry == "" && defaultRegistry != "docker.io" {
```

Reply:
I see this will update the project config with <custom-registry>/<stack>:<version> . but if this was not done and user's config stayed at <stack>:<version> then appsody will use the <custom-registry> value to pull the image anyways as that is what our default behaviour was.. am I missing something?